.travis.yml

```diff
@@ -25,8 +25,7 @@ install:
   - chmod +x /tmp/urchin/package/urchin
   - '[ -z "$WITHOUT_CURL" ] || sudo apt-get remove curl -y'
 script:
-  - if [ -n "${SHELLCHECK-}" ]; then shellcheck -s bash nvm.sh && shellcheck -s sh nvm.sh && shellcheck -s dash nvm.sh && shellcheck -s ksh nvm.sh && shellcheck -s bash install.sh ; fi
-  - if [ -z "${SHELLCHECK-}" ]; then make TEST_SUITE=$TEST_SUITE URCHIN=/tmp/urchin/package/urchin test-$SHELL ; fi
+  - if [ -n "${SHELLCHECK-}" ]; then ./shellcheck.sh ; else make TEST_SUITE=$TEST_SUITE URCHIN=/tmp/urchin/package/urchin test-$SHELL ; fi
```

Comment:
This will make `.travis.yml` not maintainable and not human readable at all especially we'll only make more files there to be tested, as the other script tests are also not separated, this will not hurt at all but only make the project much more maintainable right?